src/tools/launcher/bash_launcher.cc

```diff
@@ -30,6 +30,10 @@ static constexpr const char* BASH_BIN_PATH = "bash_bin_path";
 
 ExitCode BashBinaryLauncher::Launch() {
   wstring bash_binary = this->GetLaunchInfoByKey(BASH_BIN_PATH);
+
+  // Rlocation returns the original path if bash_binary is an absolute path.
+  bash_binary = this->Rlocation(bash_binary, true);
```

Comment:
Can the input `bash_binary` ever be just `bash.exe` for example, which would mean "whatever is on the PATH"?

Reply:
I think there are two ways to tell which shell binary to use, `BAZEL_SH` or `--shell_executable`. Both way requires user to set an absolute path. The only case `bash_binary` is a relative path (runfile path) is that when using shell toolchain (similar to python toolchain), but I'm not sure it's implemented.